extensions/spotify-controls/src/utils.ts

```diff
@@ -15,3 +15,14 @@ export async function runAppleScriptSilently(appleScript: string) {
   await closeMainWindow();
   await runAppleScript(appleScript);
 }
+
+/**
+ * Runs the AppleScript
+ *
+ * @param appleScript - The AppleScript to run
+ * @throws An error when the AppleScript fails to run
+ * @returns A promise that is resolved when the AppleScript finished running
+ */
+export async function runAppleScriptAndReturn(appleScript: string) {
+  return await runAppleScript(appleScript);
+}
```

Comment:
Do you think we need this? Seems like the function doesn't add anything on top of the `runAppleScript` one. How about keeping it simple and remove `runAppleScriptAndReturn` and use `runAppleScript` directly?

Reply:
I just copied the style I saw without thinking about it. It makes sense to just use the function directly.

Reply:
Want to make this change as well and use the function directly before I merge the PR? Please re-request a review afterwards.